test/controllers/organization_controller_test.exs

```diff
@@ -1,176 +1,86 @@
+    test "renders 403 when not authorized", %{conn: conn} do
+      assert conn |> request_create |> json_response(403)
     end
   end
 
   describe "update" do
-    @tag :authenticated
-    test "updates and renders chosen resource when data is valid", %{conn: conn, current_user: current_user} do
-      organization = insert(:organization)
-      insert(:organization_membership, organization: organization, member: current_user, role: "admin")
-
-      payload =
-        build_payload
-        |> put_id(organization)
-        |> put_attributes(@valid_attrs)
-
-      path = conn |> organization_path(:update, organization)
-      conn = conn |> put(path, payload)
-
-      assert json_response(conn, 200)["data"]["id"]
-      assert Repo.get_by(Organization, @valid_attrs)
+    @tag authenticated: :admin
+    test "updates and renders chosen resource when data is valid", %{conn: conn} do
+      assert conn |> request_create(@valid_attrs) |> json_response(201)
     end
 
-    @tag :authenticated
-    test "does not update chosen resource and renders errors when data is invalid", %{conn: conn, current_user: current_user} do
-      organization = insert(:organization)
-      insert(:organization_membership, organization: organization, member: current_user, role: "admin")
-
-      payload =
-        build_payload
-        |> put_id(organization)
-        |> put_attributes(@invalid_attrs)
-
-      path = conn |> organization_path(:update, organization)
-      conn = conn |> put(path, payload)
-
-      assert json_response(conn, 422)["errors"] != %{}
+    @tag authenticated: :admin
+    test "renders 422 when data is invalid", %{conn: conn} do
+      assert conn |> request_create(@invalid_attrs) |> json_response(422)
     end
 
-    test "does not update resource and renders 401 when not authenticated", %{conn: conn} do
-      organization = insert(:organization)
-
-      payload =
-        build_payload
-        |> put_id(organization)
-        |> put_attributes(@invalid_attrs)
-
-      path = conn |> organization_path(:update, organization)
-      conn = conn |> put(path, payload)
-
-      assert json_response(conn, 401)
+    test "renders 401 when not authenticated", %{conn: conn} do
+      assert conn |> request_create |> json_response(401)
     end
 
     @tag :authenticated
-    test "does not update resource and renders 403 when not authorized", %{conn: conn, current_user: current_user} do
-      organization = insert(:organization)
-      insert(:organization_membership, organization: organization, member: current_user, role: "member")
-
-      payload =
-        build_payload
-        |> put_id(organization)
-        |> put_attributes(@invalid_attrs)
-
-      path = conn |> organization_path(:update, organization)
-      conn = conn |> put(path, payload)
-
-      assert json_response(conn, 403)
+    test "renders 403 when not authorized", %{conn: conn} do
+      assert conn |> request_create |> json_response(403)
     end
 
```

Comment:
Just noting that we talked about this after this comment, so think change is pending.